base/core/include/motis/core/common/logging.h

```diff
@@ -106,4 +106,17 @@ struct manual_timer final {
   std::chrono::time_point<std::chrono::steady_clock> start_;
 };
 
+inline void clog_set_progress_bounds(double output_low = 0.,
```

Comment:
Ich hatte angefangen alles neue in ".cc" Dateien zu packen in der Hoffnung, Compile Times zu verringern. "Kleinvieh macht auch mist."

Reply:
wollte ich auch zuerst, aber dann habe ich gesehen, dass in der logging.h weiter oben noch mehr inline funktionen sind ...

Reply:
Broken-Windows-Effect. Macht es nicht besser.
Richtig wäre: die anderen inline Funktionen auch in ".cc" Files zu verschieben.

Reply:
k